geo_heatmap.py

```diff
@@ -63,6 +63,24 @@ def loadKMLData(self, file_name):
                 self.coordUpdate(lat_lon)
                 pb.update(i)
 
+    def load_takeout_zip_data(self, file_name):
+        """
+        Load google location data from a "takeout-*.zip" file.
+        """
+        zip_file = zipfile.ZipFile(file_name)
+        namelist = zip_file.namelist()
+        json_path = 'Takeout/Location History/Location History.json'
+        kml_path = 'Takeout/Location History/Location History.kml'

```

Comment:
The problem here is, that the files aren't always called `Location History.json` or `Location History.kml`. For example for me they are called `Standortverlauf.json` so the name depends on the language of the Google account.

Reply:
The folder is also called `Standortverlauf`.

Reply:
I'm actually not sure how to solve this

Reply:
We could look for predefined file names for known languages and then fall back to the first json or kml file under the expected directory structure

Reply:
One obvious way would be to store a list of different names of folders in a list and then performing linear search until a finds a match. One disadvantage would be that the code could look a little dirty but it will skip a step for the layman.

Reply:
In the .html file that is in the takeout archive I found this: `<h1 class="data-folder-name" data-english-name="LOCATION_HISTORY" data-folder-name="Standortverlauf">Standortverlauf</h1>` So from `data-folder-name` we get what we are looking for

Reply:
Or leave it, just use grub to locate and go to the only folder which exists inside the takeout directory and grab the json or kml which exists. Break if file not found.

Reply:
> Or leave it, just use grub to locate and go to the only folder which exists inside the takeout directory and grab the json or kml which exists. Break if file not found.

The problem with that is if someone selects more than just the location history this won't work.

Reply:
I like the idea of parsing the HTML file. In my export, that file is located at `Takeout/archive_browser.html`. @luka1199 what's the path of this file in your German language archive?

Reply:
`Takeout/Archiv_Übersicht.html`

Reply:
@luka1199 can you test https://github.com/luka1199/geo-heatmap/pull/18/commits/1cf275298347fc2b5c708980bdcc2a5825e29a07?